ingestion/pom.xml

```diff
@@ -84,6 +75,31 @@
         </executions>
       </plugin>
 
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-enforcer-plugin</artifactId>
+        <executions>
+          <!--
+            We want to keep Ingestion compatible with Java 8 runtime for Beam runners, #518.
+            Unfortunately this doesn't seem to work on Reactor modules, though:
+            https://github.com/mojohaus/mojo-parent/issues/85
```

Comment:
I hoped that this would fail before applying the fix, verifying that it could catch this problem, but it didn't. I kept this in, because if it does fail on third-party dependencies, we would probably like to know if/when it changes in the future. This rule is from `extra-enforcer-rules` which was added #451 but not actually used for anything so far, I think.